### PR TITLE
feat: add documentation to use vector index client

### PR DESCRIPTION
**Pull request by pratik151192**
Added a new page for MVI that is not supposed to be merged but shared as a PDF with customers.

### CODE REVIEW

**Comment by cprice404** [comment]
@pratik151192 please mark this as a draft, we do not intend to merge it until we have moved out of the private preview phase

File: docs/develop/sdks/python/vector-index.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+---
+sidebar_position: 1
+sidebar_label: Momento Vector Index
+sidebar_class_name: sidebar-item-python-cheat-sheet
+title: Cheat Sheet for Python + Momento Vector Index
+description: Everything to get you going with coding using Python and Momento Vector Index
+---
+
+# Cheat Sheet for Python with Momento Vector Index
+If you need to get going quickly with Python and Momento Vector Indexs, this page contains the basic API calls you'll need.
```
Comment (pete-naylor) [inline docs/develop/sdks/python/vector-index.md]: We should drop the "s" here - "Momento Vector Index,"

File: docs/develop/sdks/python/vector-index.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+def delete_items(client, index_name: str):
+    item_ids_to_delete = ["test_item_1", "test_item_3"]
+    delete_response = client.delete_item_batch(index_name, ids=item_ids_to_delete)
+    if isinstance(delete_response, DeleteItemBatch.Success):
+        print("Successfully deleted items")
+    elif isinstance(delete_response, DeleteItemBatch.Error):
+        raise(Exception("Error while deleting items " + delete_response.message))
+```
+
+## Notes of usage
```
Comment (pete-naylor) [inline docs/develop/sdks/python/vector-index.md]: "Usage notes" might be best.

**Comment by NoSQLKnowHow** [comment]
I copied the content of this PR and put it in the docs splitting PR so we'd have some MVI content. IMO, this PR can be closed. is that ok?

**Pull request closed without merging**